### DISCUSSION TITLE
Copilot not authenticating with neovim

### DISCUSSION THREAD

**Discussion by apinanyogaratnam** [community/community#75145] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm having issues with copilot generating results with neovim. 

- typing `:Copilot status` gives `Enabled and online`
- typing `:Copilot` says `Not authenticated: NotSignedIn`
- I do both `:Copilot auth` and `:Copilot auth signin` and both say `Copilot: Authenticated as GitHub user apinanyogaratnam`
- I restart neovim and try to get code completions but nothing happens
- endless cycle of all this

Is there a cache or something about my credentials or something?
Any suggestions or possible solutions to this?

/Users/myuser/.config/nvim/init.lua:
```
-- Use a protected call so we don't error out on first use
local status_ok, _ = pcall(require, "plug")
if not status_ok then
  vim.cmd [[
    set runtimepath^=~/.vim runtimepath+=~/.vim/after
    let &packpath = &runtimepath
    source ~/.local/share/nvim/site/autoload/plug.vim
  ]]
end

-- Call vim-plug commands using vim.cmd [[ ... ]]
vim.cmd [[
  call plug#begin('~/.config/nvim/plugged')

  Plug 'nvim-telescope/telescope.nvim', { 'do': ':UpdateRemotePlugins' }
  Plug 'neoclide/coc.nvim', { 'branch': 'release' }
  Plug 'nvim-lua/plenary.nvim'
  Plug 'github/copilot.vim', { 'branch': 'release' }

  call plug#end()
]]

-- Set Tab as the keymap to accept coc auto completions
vim.api.nvim_set_keymap('i', '<Tab>', 'pumvisible() ? "\\<C-n>" : "\\<Tab>"', {expr = true, noremap = true})
vim.api.nvim_set_keymap('i', '<S-Tab>', 'pumvisible() ? "\\<C-p>" : "\\<S-Tab>"', {expr = true, noremap = true})
vim.api.nvim_set_keymap('i', '<Tab>', 'pumvisible() ? coc#pum#confirm() : "\\<Tab>"', {expr = true, silent = true, noremap = true})

-- Now you can require any Lua module that you have set up
require('keymaps') -- Make sure you have a keymaps.lua file in your lua/ directory
```

/Users/myuser/.config/nvim/lua/keymaps.lua:
```local map = vim.api.nvim_set_keymap
local opts = { noremap = true, silent = true }

-- You can replace 'Cmd' with 'Ctrl' if you are not on macOS
map('n', 'ff', '<Cmd>Telescope find_files<CR>', opts)```

**Comment by pka420** [comment]
Having the same issue in neovim 0.9.6 and mac M1

**Reply by pka420** [reply]
Hey, fixed it, 
I realized that you have to also sign up to get access to copilot and not just authorize neovim to your account. Stupid me.

**Comment by kawakami-o3** [comment]
`:Copilot auth signout`
or delete `~/.config/github-copilot/hosts.json` .

https://github.com/zbirenbaum/copilot.lua/issues/169

**Reply by vikks** [reply]
👍🏽
